shared/src/vyxal/VAny.scala

```diff
@@ -134,11 +139,21 @@ end extension
 
 case class VConstructor(
     name: String
-)
+):
+  override def toString = s"$name constructor"
 
 case class VObject(
     className: String,
     fields: Map[String, (Visibility, VAny)],
-)
+):
+  override def toString =
+    val fs = fields.map {
+      case (name, (vis, value)) => s"$value ${vis match
```

Comment:
I'd kinda prefer to have the match outside the string, assigned to a variable `prefix` or something